docs/getting-started/app-custom-theme.md

```diff
@@ -273,6 +273,115 @@ const LogoFull = () => {
 };
 ```
 
+## Custom Icons
+
+You can also customize the Project's _default_ icons.
+
+We can change the following icons:
```

Comment:
I would just make this a link to the code where we have the default icons that way it stays up to date: https://github.com/backstage/backstage/blob/master/packages/app-defaults/src/defaults/icons.tsx

Reply:
Perfect!